clicommand/pipeline_upload.go

```diff
@@ -21,7 +21,7 @@ import (
 
 var PipelineUploadHelpDescription = `Usage:
 
-   buildkite-agent pipeline upload <file> [arguments...]
+   buildkite-agent pipeline upload [<file>] [arguments...]
```

Comment:
I think the square brackets should replace the pointy ones? Rather than being additional.

```suggestion
   buildkite-agent pipeline upload [file] [arguments...]
```

Reply:
There's a couple of others with both which I guess should also be fixed:

```
pda@paulbookpro ~/bk/agent ❯ rg '\[<'
clicommand/meta_data_set.go
16:   buildkite-agent meta-data set <key> [<value>] [arguments...]

clicommand/annotate.go
18:   buildkite-agent annotate [<body>] [arguments...]
```

Reply:
Yeah I was wondering about those brackets! Cool, I'll update all those places.